### ISSUE TITLE
Placement: Standard cell placement is not correct which leads to LVS issues

### ISSUE THREAD

**Issue by lulu9312** [The-OpenROAD-Project/OpenROAD#2541]
I am working on OpenFASOC' s ldo-gen which has a comparator (cmp1). The cmp1 is placed in seperate voltage domain.Now the issue for one of the setting is that the VDD rail is passing over cmp1's VSS pin and VSS rail is passing over cmp1's VDD pin this then further causes LVS issue.
Here is screenshots for incorrect placement 
Incorrect placement :
![cmp1_error](https://user-images.githubusercontent.com/61928883/203491869-d16d74a4-0ee4-43a5-8d90-67b4e2f9a308.jpg)

Here is a screenshot of correct placement
![cmp1_correct](https://user-images.githubusercontent.com/61928883/203492952-4e1d5ebd-d6ff-43d1-80c9-0911d09fbdfe.png)

OpenROAD version:
OpenROAD v2.0-5170-gd919ca2d4 

To reproduce the issue:
[detail_place_ldo_sky130hvl_base_2022-11-23_12-44.tar.gz](https://github.com/The-OpenROAD-Project/OpenROAD/files/10073446/detail_place_ldo_sky130hvl_base_2022-11-23_12-44.tar.gz)

**Comment by vijayank88** [comment]
@lulu9312 
Please follow issue template to fill `OpenROAD` version, OS etc.,

**Comment by vijayank88** [comment]
with latest OR LDO placement as follows
![OR_2541](https://user-images.githubusercontent.com/85787913/203512630-dde2e281-fe95-401d-b589-f4ce15183ed8.png)

**Comment by lulu9312** [comment]
> with latest OR LDO placement as follows ![OR_2541](https://user-images.githubusercontent.com/85787913/203512630-dde2e281-fe95-401d-b589-f4ce15183ed8.png)

This is incorrect,this is same as the error image above.

**Comment by rovinski** [comment]
This is a double-height standard cell legalization issue. The issue could be in either DPL or DPO.

**Comment by maliberty** [comment]
OR doesn't support double height cells.  I suggest you manually place it and set its status to firm.

**Comment by rovinski** [comment]
It used to support multi-height cells. Was that completely removed? Would it take significant effort to support it again?

**Comment by lulu9312** [comment]
> OR doesn't support double height cells. I suggest you manually place it and set its status to firm.

@maliberty Yes. I thought of that but I have kept the status to PLACED as keeping it FIRM causes site aligned failed issue.
This is implemented here https://github.com/idea-fasoc/OpenFASOC/pull/140

**Comment by rovinski** [comment]
@lulu9312 then you need to make sure you snap the cell to a cell site when you place it.

**Comment by lulu9312** [comment]
@rovinski Sorry , but what do you mean by snap the cell ?

**Comment by maliberty** [comment]
@rovinski it used to incorrectly attempt to support multi-height cells only in dpl.  Since that code was not in use and not at all correct it was removed.  Creating proper support is a project that would need non-trivial effort.

When manually placing the cell you must pick a location that aligns with the sites/rows.

**Comment by rovinski** [comment]
@lulu9312 There is a cell site grid in each row. You need to manually align your cell to this grid, otherwise fill cells cannot be inserted properly.

**Comment by lulu9312** [comment]
@maliberty @rovinski Understood.

**Comment by maliberty** [comment]
There have been substantial improvments to mult-height cells since 2022.  Please reopen if still an issue.
